Clinical-Genomics/scout
# Patch release 4.21.2 bump

This PR marks a new Scout release. We apply semantic versioning. This is a patch release since it only fixes existing functionality.

**How to test**:
1. previously tested on merged branches, but deploy to stage and check that things appear normal

**Expected outcome**:
The functionality should be working

**Review:**
